code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -198,8 +198,7 @@
 		else if(M.getFireLoss())
 			M.adjustFireLoss(-reac_volume)
 			if(show_message)
-				M << "<span class='danger'>You feel your burns healing! It stings like hell!</span>"
-			M.emote("scream")
+				M << "<span class='danger'>You feel your burns healing!</span>"
```

Comment:
You need to add a visible_message so that other people have feedback of the burns healing.

Thanks